### ISSUE TITLE
Add optional ability to translate extracted and possibly edited text via some online translation tool.

### ISSUE THREAD

**Issue by Tsippaduida** [skvark/Textractor#11]

**Comment by skvark** [comment]
I did actually some investigating on translation support before I started writing this app. Best case would be to be able to use some pre-existing native SFOS translation application to translate the text. This requires some invoking api (possibly via dbus) and I don't know is it allowed in the Jolla store. Other solution is to use Google translate or some other limited (= non free or rate limited) api.

**Comment by Tsippaduida** [comment]
I agree. I can live without translation, but having it available would be real nice. Offline solution would be best. Google and other web translations can be utilized already, but requires bit work.
